### ISSUE TITLE
touch device is not rotating correctly with screen

### ISSUE THREAD

**Issue by opennomad** [JeanSchoeller/iio-hyprland#21]
I'm running on EndeavrousOS (rolling distro, fully current) with Hyprland 0.41.0-1 and the AUR version of iio-hyprland from git which reports as version r49.f31ee41-1.

I'm on a HP x360 spectre and the screen is rotating beautifully in every direction as expected using the following line in the config:

```
monitor=eDP-1,preferred,auto,2,transform,0
```

The issue I have is that the touch device doesn't seem to rotate as I think it should. When I use the following config:

```
input {
  touchdevice {
    output = eDP-1
    transform = 2
  }
}
```

it forces the rotation based on the value for the `transform`, rather than following the rotation of the screen. So for most cases `0` is the right value, but i've not been able to make it match the rotation of the screen automatically.

Is there something I'm missing to make that happen?

**Comment by JeanSchoeller** [comment]
#Could you try building the package from Github and not the AUR. Just to know if it could be a version mismatch from my behalf.
I believe this has been addressed in #20

**Comment by opennomad** [comment]
:boom: that was it. the version installed from git clone works correctly. Thank you!
